readme.md

```diff
@@ -56,7 +56,21 @@ The thing you want to open. Can be a URL, file, or executable.
 
 Opens in the default app for the file type. Eg. URLs opens in your default browser.
 
-#### app
+#### options
+
+Type: `object`
+
+An optional options hash.
+
+##### options.wait
+
+Type: `boolean`
+
+Defaults to `true` and means that opn will wait calling the callback (if one is given) until the app have been closed again. If set to `false` the callback will be called immediately after the child process have spawned.
+
+Note that waiting is only possible if the app name is either `darwin` or `win32`.
```

Comment:
It's because of this set of if-sentences in the main function:
https://github.com/sindresorhus/opn/blob/a20ea1810c2805b3126134042a8ca9841284aa48/index.js#L24-L65

It seems that the support to actually wait is only available for either `darwin` or `win32` platforms - at least it only seem to be implemented for those two